### PR TITLE
Rebrand- name

### PR DESCRIPTION
**Pull request by jjw24**
Rebranding the name to Flow Launcher:

1. Placeholders and project names- Flow.Launcher
2. Code- FlowLauncher
3. Description text: Flow Launcher

License not modified

@SysC0mp @Kalaschnik @theClueless @NickSeagull just done a first cut of the rebranding- renaming the launcher name. If you are free, please take a look and let me know if I missed anything.

### CODE REVIEW

**Comment by NickSeagull** [comment]
Looking great! Why the `psd` files though? If they are logos etc... Perhaps we should just upload the SVGs?

Also, is there a way to see them rendered?

**Comment by jjw24** [comment]
> Looking great! Why the `psd` files though? If they are logos etc... Perhaps we should just upload the SVGs?
> 
> Also, is there a way to see them rendered?

good pick up. No idea lol. I will see what's up with them later.

Have you gotten the invite to join the Flow Launcher org ?

**Comment by jjw24** [comment]
> Looking great! Why the `psd` files though? If they are logos etc... Perhaps we should just upload the SVGs?

Not sure why they are changed, reverted them back to master branch's version

**Comment by NickSeagull** [comment]
> @NickSeagull Have you gotten the invite to join the Flow Launcher org ?
Yep!

**Comment by jjw24** [comment]
@NickSeagull sorry bud, not yet ready. Still fine tuning a bit. Will let you know when done.

**Comment by jjw24** [comment]
@NickSeagull @SysC0mp @theClueless this PR is in good stage to go in. If you have time could you take a look at the new commits and approve if good to go

File: JsonRPC/wox.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 import sys
 import inspect
 
-class Wox(object):
+class Flow.Launcher(object):
```
Comment (SysC0mp) [inline JsonRPC/wox.py:7]: I think this dot still needs to be removed. I wanted to change & push that but it seems that i have no permissions to do it.

File: JsonRPC/wox.py [hunk 2]
```diff
@@ -42,74 +42,74 @@ def debug(self,msg):
         print("DEBUG:{}".format(msg))
         sys.exit()
 
-class WoxAPI(object):
+class Flow.LauncherAPI(object):
```
Comment (SysC0mp) [inline JsonRPC/wox.py:45]: Same as above

Comment (SysC0mp) [reply JsonRPC/wox.py:45]: Apart from that, all the other changes look fine to me. 👍

File: JsonRPC/wox.py (hunk 1 above)
Comment (jjw24) [reply JsonRPC/wox.py]: good catch. Cheers

**Pull request merged**